lectures/more_julia/generic_programming.md

```diff
@@ -304,15 +304,15 @@ For example, to simulate $x_{t+1} = a x_t + b \epsilon_{t+1}$ where
 $\epsilon \sim D$ for some $D$, which allows drawing random values.
 
 ```{code-cell} julia
-function simulateprocess(x₀; a = 1.0, b = 1.0, N = 5, d::Sampleable{Univariate,Continuous})
-    x = zeros(typeof(x₀), N+1) # preallocate vector, careful on the type
+function simulateprocess(x₀; a = 1.0, b = 1.0, N = 5, d::Sampleable{Univariate, Continuous})
```

Comment:
I would globally replace the `x₀` with `x_0` here.  Maybe add it to the unicode replacement list.